uwblueprint/abtc
# Authenticate Endpoints

## Notion ticket link

[Authenticate endpoints](https://www.notion.so/uwblueprintexecs/Authenticate-End-Points-71244ad4da754bee892a52d48fd75f82?pvs=4)



## Implementation description
* Changes authorization to solely check for valid token for now
* For all endpoints, we check if there is this valid token using auth middleware



## Steps to test
To test, you can attempt to access the endpoints while logged in and logged out, you should only be able to access if you're logged in and have a valid token.



## What should reviewers focus on?
* 


## Checklist
